.changeset/early-rice-tell.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I'm not aware of where these logos are used, is it ok that we're skipping a version bump in this case?

Reply:
I believe so!